release.sh

```diff
@@ -1,16 +0,0 @@
-set -e
```

Comment:
Yes!!!

An idea: instead of deleting this file - can it print instructions how to build flow? It would be nice to people who are used to this method. Or just document it in the readme.

Reply:
Will do! I think the "print instructions" way is good. Also probably we should exit with something other than 0. I'm not aware of anything which uses this script, but...

Reply:
You are right, nothing really uses this script. But some people might be used to it - it is good to give them immediate info what to do instead